L1Trigger/L1TMuonEndCap/src/PtAssignmentEngine2017.cc

```diff
@@ -24,9 +24,11 @@ float PtAssignmentEngine2017::scale_pt(const float pt, const int mode) const {
   // TRG       = 1.2*XML / (1 - 0.015*XML)
   // TRG / XML = 1.2 / (1 - 0.015*XML)
 
-  // First "physics" LUTs for 2017, deployed June 7
-  if (ptLUTVersion_ >= 6) {
-    pt_xml = fmin(20., pt);  // Maximum scale set by muons with XML pT = 20 GeV (scaled pT ~35 GeV)
+  if (ptLUTVersion_ > 7) {   // First "physics" LUTs for 2022, will be deployed in June 2022
```

Comment:
My impression (it can be wrong) is that it would better jump to eyes which is the first LUT version to be considered here if you replace it with
```suggestion
  if (ptLUTVersion_ >= 8) {   // First "physics" LUTs for 2022, will be deployed in June 2022
```
as you do in the other case for versions >= 6

If you agree, you can update (here and below). If not, just tell us, and this PR is otherwise ready to get merged